invenio/base/scripts/database.py

```diff
@@ -86,6 +86,9 @@ def init(user='root', password='', yes_i_know=False):
             (cmd_prefix + '-e "GRANT ALL PRIVILEGES ON '
              '{CFG_DATABASE_NAME}.* TO {CFG_DATABASE_USER}@localhost '
              'IDENTIFIED BY {CFG_DATABASE_PASS}"'),
+            (cmd_prefix + '-e "GRANT ALL PRIVILEGES ON '
+             '{CFG_DATABASE_NAME}.* TO {CFG_DATABASE_USER}@\'%\' '
+             'IDENTIFIED BY {CFG_DATABASE_PASS}"'),
```

Comment:
We should avoid putting Docker-specific code into the Invenio core (aka Python code) and should stay with separate Dockerfile + scripts. There are plenty other deployment methods out there that might get used by our users. Ref. [Hollywood principle](https://en.wikipedia.org/wiki/Hollywood_principle)